x-pack/legacy/plugins/alerting_ui/np_ready/public/application/sections/actions_list/components/actions_list.tsx

```diff
@@ -0,0 +1,265 @@
+        {
+          defaultMessage: 'Action Type',
+        }
+      ),
+      sortable: true,
+      truncateText: true,
+    },
+    {
+      field: 'config',
+      name: i18n.translate(
+        'xpack.alertingUI.sections.actionsList.actionsListTable.columns.config',
+        { defaultMessage: 'Config' }
+      ),
+      sortable: false,
+      truncateText: false,
+    },
+    {
+      name: i18n.translate(
+        'xpack.alertingUI.sections.actionsList.actionsListTable.columns.actions',
+        { defaultMessage: 'Actions' }
+      ),
+      actions: [
+        {
+          enabled: () => canDelete,
+          name: i18n.translate(
+            'xpack.alertingUI.sections.actionsList.actionsListTable.columns.actions.deleteActionName',
+            { defaultMessage: 'Delete' }
+          ),
+          description: canDelete
+            ? i18n.translate(
+                'xpack.alertingUI.sections.actionsList.actionsListTable.columns.actions.deleteActionDescription',
+                { defaultMessage: 'Delete this action' }
+              )
+            : i18n.translate(
+                'xpack.alertingUI.sections.actionsList.actionsListTable.columns.actions.deleteActionDisabledDescription',
+                { defaultMessage: 'Unable to delete actions' }
+              ),
+          type: 'icon',
+          icon: 'trash',
+          onClick: (item: Data) => deleteItems([item]),
+        },
+      ],
+    },
+  ];
+
+  return (
+    <section data-test-subj="actionsList">
+      <Fragment>
+        <EuiSpacer size="m" />
+        <ActionsContext.Provider value={{}}>
+          <EuiInMemoryTable
+            loading={isLoadingActions || isLoadingActionTypes || isDeletingActions}
+            items={data}
+            sorting={true}
+            itemId="id"
+            columns={actionsTableColumns}
+            rowProps={() => ({
+              'data-test-subj': 'row',
+            })}
+            cellProps={() => ({
+              'data-test-subj': 'cell',
+            })}
+            data-test-subj="actionsTable"
+            pagination={true}
+            selection={{
+              onSelectionChange(updatedSelectedItemsList: Data[]) {
+                setSelectedItems(updatedSelectedItemsList);
+              },
+            }}
+            search={{
+              filters: [
+                {
+                  type: 'field_value_selection',
+                  field: 'actionTypeId',
+                  name: i18n.translate(
+                    'xpack.alertingUI.sections.actionsList.filters.actionTypeIdName',
+                    { defaultMessage: 'Action Type' }
+                  ),
+                  multiSelect: 'or',
+                  options: Object.values(actionTypesIndex || {})
+                    .map(actionType => ({
+                      value: actionType.id,
+                      name: actionType.name,
+                    }))
+                    .sort((a, b) => {
```

Comment:
if `a` and `b` are strings, you can use `a. localeCompare(b)`

Reply:
First time hearing about `localeCompare`, pretty cool! Fixed in https://github.com/elastic/kibana/pull/47843/commits/74687508fabf277c757e82a46b33a1afec4291d8.